### ISSUE TITLE
Exception on $routePrivder.otherwise processing

### ISSUE THREAD

**Issue by fedotxxl** [ajoslin/angular-mobile-nav#22]
When "otherwise" rule is used following exception occurs:

``` javascript
TypeError: Cannot read property 'reverse' of undefined
    at changePage (http://localhost:8080/mobile-nav.js:235:71)
    at transitionStart (http://localhost:8080/mobile-nav.js:253:22)
    at Object.$get.e.$broadcast (http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.min.js:88:281)
    at navigate (http://localhost:8080/mobile-nav.js:133:16)
    at defaultRouteSuccess (http://localhost:8080/mobile-nav.js:148:5)
    at http://localhost:8080/mobile-nav.js:152:49
    at Object.$get.e.$broadcast (http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.min.js:88:281)
    at $get.ya.params (http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.min.js:80:114)
    at k.promise.then.i (http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.min.js:76:119)
    at k.promise.then.i (http://ajax.googleapis.com/ajax/libs/angularjs/1.0.3/angular.min.js:76:119) angular.min.js:60
```

$route.current.$route is undefined in this case

**Comment by ajoslin** [comment]
Hi @fedotxxl! You're right, I need to fix this.  If it goes to an invalid $route it breaks.

**Comment by ajoslin** [comment]
Fixed in 255a97f23b2bbd30592d693252c7fbd08cff7a76.

It's getting scary how much we need unit tests!!  So many cases to keep in mind.
